src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
+        Ball ball2 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
+        Ball ball3 = new Lottery().getRandomBall(new ColorSupplier().getRandomColor());
```

Comment:
`getRandomBall()` should not have any parameters
```suggestion
        Ball ball1 = new Lottery().getRandomBall();
        Ball ball2 = new Lottery().getRandomBall();
        Ball ball3 = new Lottery().getRandomBall();
```